### ISSUE TITLE
[Missing] - VPC 2.0 parameter

### ISSUE THREAD

**Issue by davidac0** [vultr/ansible-collection-vultr#110]
Would it be possible to add a VPC2.0 parameter for `vultr.cloud.instance`? At the moment there is just `vpcs` which only supports v1.

Thanks & Kind regards,
David

**Comment by resmo** [comment]
Hi @davidac0, I did an implementation in #118, but while testing I saw the same unexpected behavior of the API that vultr engineers are currently investigating. This is the reason why it is on hold.

**Comment by nekinie** [comment]
I assume we are still waiting on Vultrs investigation?

**Comment by resmo** [comment]
@jasites do you have any updates about this?

**Comment by jasites** [comment]
> @jasites do you have any updates about this?

I don't have any new information on this issue but I will be devoting some time in the near future to investigating further and working towards a resolution here.

**Comment by resmo** [comment]
> VPC 2.0 is now deprecated and all focus for further development is being directed to what was known as VPC v1

oh... that came unexpected. Looking forward to deprecated vpc2 in code.
